components/tooltip/tooltip.js

```diff
@@ -470,7 +470,10 @@ class Tooltip extends RtlMixin(LitElement) {
 			<div class="d2l-tooltip-container">
 				<div class="d2l-tooltip-position" style=${styleMap(tooltipPositionStyle)}>
 					<div class="d2l-body-small d2l-tooltip-content">
-						<slot></slot>
+						<div role="text">
+							<!-- Note: role="text" is a workaround for Safari. Otherwise, list-item content is not announced with VoiceOver" -->
```

Comment:
I'd be inclined to move this comment into JavaScript above, since the HTML comment is so readily seen (by users) inspecting the DOM.  I like the comment, just rather it be more hidden from view.